### DISCUSSION TITLE
How to install postgresql_extensions on a running cluster?

### DISCUSSION THREAD

**Discussion by jwcfree** [autobase-tech/autobase#221] [General]
Hello guys! I want to install some plugins, like pg_cron, pg_partman on a running cluster with ansible role.
I edited vars/main.yml and selected this plugins in the block postgresql_extensions... But an error occurred:
`"Whoops! data directory /var/lib/pgsql/14/data is already initialized"`
How to install extensions correctly?

**Comment by vitabaks** [comment]
Please use tags https://github.com/vitabaks/postgresql_cluster/blob/master/tags.md


`ansible-playbook deploy_pgcluster.yml --tags postgresql_extensions`

**Reply by vitabaks** [reply]
Added a new playbook `config_pgcluster.yml` designed to update configurations after cluster deployment (https://github.com/vitabaks/postgresql_cluster/pull/218).

**Reply by jwcfree** [reply]
Nice work!!! Thank you 🙏

**Comment by jwcfree** [comment]
I would also like to know about extesions path. Maybe I missed something, but error:
```
TASK [postgresql-extensions : Add extensions to the databases]
Extension pg_cron is not available
```

**Reply by vitabaks** [reply]
Are you sure you have installed the expansion package?

Check that it is listed in vars/Debian.yml (or vars/RedHat.yml)

example:

```
postgresql_packages:
   ...
  - postgresql-{{ postgresql_version }}-cron
```

next, launch a playbook with tags:

`ansible-playbook deploy_pgcluster.yml --tags install_postgres, postgresql_extensions`
